### DISCUSSION TITLE
Want to support other language (korea)

### DISCUSSION THREAD

**Discussion by uknowpro** [SnosMe/awakened-poe-trade#890] [Q&A]
Hi, I am a software enginner and a player of poe.
Could I release this program to Korean poe gamers after adding korean language version? It is not for commercial purposes.

**Comment by SnosMe** [comment]
Yes, you can. But leave the link here for your fork.

**Comment by uknowpro** [comment]
Thanks. I forked, but haven't started yet. Looks like I'll start working on it little by little soon. The goal is to release it before the 3.21 (poe).

here: https://github.com/uknowpro/awakened-poe-trade-for-korean
